src/darwin/Framework/CHIP/MTRStorage.h

```diff
@@ -20,11 +20,13 @@
 NS_ASSUME_NONNULL_BEGIN
 
 /**
- * The protocol definition for the CHIPPersistenStorageDelegate
+ * This protocol is used by the Matter framework to read and write storage.
  *
- * All delegate methods will be called on the supplied Delegate Queue.
+ * All storage methods will be called on a single internal work queue (so the
```

Comment:
Nit: We could just say something like "The Matter framework will not call storage methods concurrently." instead of referring to a dispatch queue.

Reply:
Well, there are two separate things I wanted to communicate here:

1) The Matter framework will not call storage methods concurrently.
2) The callee must not make any assumptions about which thread it's on when the storage methods get called.

I suppose I could just say that explicitly?

Reply:
Done.